src/test/fuzz/partially_downloaded_block.cpp

```diff
@@ -50,7 +51,8 @@ FUZZ_TARGET(partially_downloaded_block, .init = initialize_pdb)
         return;
     }
 
-    CBlockHeaderAndShortTxIDs cmpctblock{*block};
+    FastRandomContext fast_random_context{ConsumeUInt256(fuzzed_data_provider)};
+    CBlockHeaderAndShortTxIDs cmpctblock{*block, fast_random_context.rand64()};
```

Comment:
No need for the actual rng, we can just consume the nonce from the fuzz data.

```suggestion
    CBlockHeaderAndShortTxIDs cmpctblock{*block, fuzzed_data_provider.ConsumeIntegral<uint64_t>()};
```